### PR TITLE
Merge back release notes from production

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

**Review by mluypaert — APPROVED** [review]

**Comment by markquintontulloch** [comment]
@mluypaert UI tests are failing because the fix Adam put in (pushing package-lock.json) hasn't made it to beta yet.  What's the procedure here?  I assume it's as follow:

1. create a hotfix branch for beta as v0.25.0-rc2 (presumably we don't need to hotfix production as the deployment will bring that change across)
2. deploy the beta hotfix
3. close this PR and create a new PRmerge/production branch to sort out the releases notes
4. carry on with deployment to production by creating release/v0.25.0 branch

**Comment by mluypaert** [comment]
> @mluypaert UI tests are failing because the fix Adam put in (pushing package-lock.json) hasn't made it to beta yet. What's the procedure here? I assume it's as follow:
> 
>     1. create a hotfix branch for beta as v0.25.0-rc2 (presumably we don't need to hotfix production as the deployment will bring that change across)
> 
>     2. deploy the beta hotfix
> 
>     3. close this PR and create a new PRmerge/production branch to sort out the releases notes
> 
>     4. carry on with deployment to production by creating release/v0.25.0 branch

@markquintontulloch Nearly. You don't need to deploy to the beta environment if the changes made to fix the UI integration test were do not affect any curation software functionality (like only changes to the ui intergration test target in the Makefile, or only changes to the github-actions workflow files), and you can keep the current PR and just merge beta into it after fixing beta, before merging the PR into beta.

So the procedure for getting this verify-test-ui integration test fixed and be able to continue with the prod deployment would be:

1. Create a hotfix branch from `beta` as `release/v0.25.0-rc2` (we indeed don't need to hotfix production as the deployment will bring that change across) and cherry-pick and required changes onto it (as to not include other changes from alpha which would be intended for the next release).
2. Open a PR to merge the `release/v0.25.0-rc2` branch into the `beta` branch and merge it on approval and successful integration tests.
3. Merge the `beta` branch into the `PRmerge/production` branch (the one in this PR), wait till the integration tests pass and then merge this PR (into `beta`).
4. Deploy beta as `v0.25.0-rc2` if the integration test fix introduced any changes requiring deployment & testing and test.
5. Carry on with deployment to production by creating the `release/v0.25.0` branch (from `beta`).

**Comment by markquintontulloch** [comment]
👍 Thanks @mluypaert

**Pull request merged**
